0000-template.md

```diff
@@ -1,51 +1,48 @@
+- Title: (fill me in with a human-focussed title for the RFC)
 - Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
-- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- RFC PR: [#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Tracking Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- Team: (fill me in with the team or teams responsible for this RFC)
+- Keywords: (fill me in with keywords which may be useful when searching for this RFC)
```

Comment:
I don't think we need this as a metadata field at the very top. We can just suggest in another section to add any other terms useful for people to search for.

Reply:
The point in having it in metadata is to make it visible to tools (both before and after RFCs are accepted). I hope that we can use them to 'search by tag' rather than just searching for free text.

I did suggest we might want to move all metadata elsewhere in the document, do you prefer that?

Reply:
What tools do you have in mind, besides search engines and grep, both of which can find keywords anywhere in the document?

I'm also hoping any search *we* add can similarly find keywords anywhere in the document.

Anything up at the top of the document is more stuff people have to skim over before they reach the summary and other descriptive text.

Reply:
I have two tools in mind: one is a bot that can set the team and labels on the PR from the team and keywords in the RFC text, thus relieving some need for triage. The second is an enhanced RFC book site, where one could browse RFCs with a given tag/keyword, see the keywords in an index of RFCs, as well as search explicitly for a keyword (e.g., there is a huge difference in the word 'trait' occuring in the text of an RFC and the word being an identified keyword, the latter implies this is an RFC about the trait system, the former could just be a casual mention). https://www.ncameron.org/rfcs/ is a prototype of this idea.

> Anything up at the top of the document is more stuff people have to skim over before they reach the summary and other descriptive text.

Do you prefer having metadata at the end of the doc? Or in a horizontal table like Ember RFCs? Or after the summary? I totally agree that it's sub-optimal to have the metadata at the top, but wasn't sure which alternative to propose.